core/src/main/java/hudson/PluginWrapper.java

```diff
@@ -1071,6 +1071,27 @@ public boolean hasImpliedDependents() {
         return false;
     }
 
+    /**
+     * Get list of implied dependencies.
+     * @since 2.292
```

Comment:
normally updated in batch, no guarantees on a release version

```suggestion
     * @since TODO
```